pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -55,14 +55,15 @@ func NewAutoScalerManager(
 	kubeInformerFactory kubeinformers.SharedInformerFactory,
 	recorder record.EventRecorder) *autoScalerManager {
 	tcLister := informerFactory.Pingcap().V1alpha1().TidbClusters().Lister()
-	stsLister := kubeInformerFactory.Apps().V1().StatefulSets().Lister()
 	return &autoScalerManager{
 		kubecli:   kubecli,
 		cli:       cli,
 		tcLister:  tcLister,
 		tcControl: controller.NewRealTidbClusterControl(cli, tcLister, recorder),
+		pdControl: pdapi.NewDefaultPDControl(kubecli),
 		taLister:  informerFactory.Pingcap().V1alpha1().TidbClusterAutoScalers().Lister(),
-		stsLister: stsLister,
+		tmLister:  informerFactory.Pingcap().V1alpha1().TidbMonitors().Lister(),
```

Comment:
Is this used?

Reply:
Will be used when creating or deleting clusters to update tidbmonitors

Reply:
Why do we need to update tidbmonitors in the autoscaler controller?

Reply:
Because we need to add the created clusters to the monitor so that PD can fetch metrics